src/domains/job/recruitment/components/WriteRecruitmentPostingForm.jsx

```
@@ -10,24 +10,30 @@ import ClosingDate from './form/ClosingDate'
 import CompanyWebsite from './form/CompanyWebsite'
 import WorkPlace from './form/WorkPlace'
 import Experience from './form/Experience'
+import useAuthStore from '../../../../store/login/useAuthStore'
 
-const WriteRecruitmentPostingForm = () => {
+const WriteRecruitmentPostingForm = ({ defaultValues, onSubmit }) => {
```

ESLint

[error] 'defaultValues' is missing in props validation

(react/prop-types)